temporalcli/commands.env.go

```diff
@@ -8,45 +8,75 @@ import (
 	"github.com/temporalio/cli/temporalcli/internal/printer"
 )
 
+func (c *TemporalEnvCommand) envNameAndKey(cctx *CommandContext, args[] string, keyFlag string) (string, string, error) {
+	if len(args) > 0 {
+		cctx.Logger.Warn("Arguments to env commands are deprecated; please use -E and/or -k instead.")
+
+		if (c.Parent.Env != "default" || keyFlag != "") {
+			return "", "", fmt.Errorf("cannot specify both an argument and -E/-k; please use -e and/or -k")
```

Comment:
Should not assume user is using aliases. And, as often happens with these aliases, this statement is unclear what each is referring to and there's an typo (i.e. `-e` does not exist).

Reply:
Oops, thought I caught all the `-e`s when I changed to `-E`—will fix.

I don't think any assumption is being made here—we're _telling_ the user to use `-E` and `-k`. If this is confusing (which it probably will not be for most people, since they can guess that `E` stands for `environment`), they can always look at `--help`.